### ISSUE TITLE
New query parameter for GET List workflow runs API

### ISSUE THREAD

**Issue by valpet93** [Shuffle/Shuffle#1507]
As user I want query a specific workflow, which is executed due to a subflow trigger, using a execution_parent id in order to retrieve its result.

A solution might be a new query parameter (execution_parent) on the existing GET List workflow runs API 

![image](https://github.com/user-attachments/assets/580d77f7-4917-4f3f-bea7-d8ef6063a496)

**Comment by frikky** [comment]
> As user I want query a specific workflow, which is executed due to a subflow trigger, using a execution_parent id in order to retrieve its result.
> 
> A solution might be a new query parameter (execution_parent) on the existing GET List workflow runs API
> 
> ![image](https://private-user-images.githubusercontent.com/34856348/369456488-580d77f7-4917-4f3f-bea7-d8ef6063a496.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwMDM5MDksIm5iZiI6MTcyNzAwMzYwOSwicGF0aCI6Ii8zNDg1NjM0OC8zNjk0NTY0ODgtNTgwZDc3ZjctNDkxNy00ZjNmLWJlYTctZDhlZjYwNjNhNDk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIyVDExMTMyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0MmRhZjc5NzhkZGU2Yzk0YmY2ZDFkYjY1ZjMzNWFmNTk4NDJkNDQ4YjExNmQzZjI0OTAzZDhhYTNkODIyNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.f-PUyGtJ0UitR1hqwRddGPRH2fsrm6JL2WaBr0E5DbM)

Good idea! If we were to expand this, we would most likely expand the Execution Search API however: `/api/v1/workflows/search`

For our v2 API, we may move all execution queries to it, as it supports pagination, search types (status/timestamps) etc. Adding execution parent to that one wouldn't be too much work :)
